### PR TITLE
BFB-375: Removed Shapes

### PR DESCRIPTION
**Pull request by m4rtint**
**JIRA Ticket**
https://perigongames.atlassian.net/browse/BFB-375

**Description**
- Removed Shapes and everything associated -  Checkpoints.

### CODE REVIEW

File: Assets/_BForBoss/_Character/FPS_Camera_Blends.asset [hunk 1]
```diff
@@ -35,47 +35,3 @@ MonoBehaviour:
         m_PreInfinity: 0
         m_PostInfinity: 0
         m_RotationOrder: 0
-  - m_From: CM Crouch
```
Comment (Gursaanj) [inline Assets/_BForBoss/_Character/FPS_Camera_Blends.asset:38]: Is this change related to Shapes?

File: ProjectSettings/EditorBuildSettings.asset [hunk 2]
```diff
@@ -20,9 +17,6 @@ EditorBuildSettings:
   - enabled: 1
     path: Assets/_BForBoss/_Weapons/Scenes/GunRange.unity
     guid: bc624f0b2db8b7d4286c919435fb2f98
-  - enabled: 1
-    path: Assets/_BForBoss/_LevelDesign/Scenes/CKJ_Weapon_Sandbox.unity
```
Comment (Gursaanj) [inline ProjectSettings/EditorBuildSettings.asset:24]: Was this scene meant to be removed from the editor settings?

File: Assets/_BForBoss/_Character/FPS_Camera_Blends.asset (hunk 1 above)
Comment (m4rtint) [reply Assets/_BForBoss/_Character/FPS_Camera_Blends.asset:38]: This was a side fix. Removed all the blends related to 3rd person.

File: ProjectSettings/EditorBuildSettings.asset (hunk 2 above)
Comment (m4rtint) [reply ProjectSettings/EditorBuildSettings.asset]: Woops, adding that back in.

**Review by Gursaanj — APPROVED** [review]
